tests/pos/CollectionStrawMan3.scala

```diff
@@ -114,13 +114,13 @@ object CollectionStrawMan1 {
   /* --------- Concrete collection types ------------------------------- */
 
   /** Concrete collection type: List */
-  sealed trait List[+A] extends Seq[A] with FromIterator[List] {
+  sealed trait List[type +Elem] extends Seq[Elem] with FromIterator[List] {
```

Comment:
Does that mean that `TypeComparer#compareAliasedRefined` is no longer needed?
It uses this as an example:
```
trait Lambda[type Elem]
trait Lst[T] extends Lambda[T]
```
and if I understand this commit correctly, this is no longer allowed